nimbleclient/query.py

```diff
@@ -0,0 +1,86 @@
+#
+#   © Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+
+
+def and_(*args):
+    ''' Build AND filters '''
```

Comment:
Mostly a linting thing. Use `"""` not `'''` for docstrings, and don't leave a space between these characters and the first / last words.

Reply:
Agreed, I will update it accordingly.